service/routes.py

```diff
@@ -89,6 +89,22 @@ def create_wishlists():
         status.HTTP_201_CREATED
     )
 
+######################################################################
+# List ALL WISHLISTS FOR A USER
+######################################################################
+@app.route("/wishlists/<int:user_id>", methods=["GET"])
+def list_wishlists_by_userid(user_id):
+    app.logger.info("Request for wishlists with user_id: %s", user_id)
+    res = Wishlist.find_all_by_user_id(user_id)
+    if not res:
```

Comment:
probably you need to change ```if not res``` to ```if len(res) == 0``` because the orm will never return a None. When there is no such wishlist corresponding to a user_id, the orm will return an empty list.